storage/rocksdb/build_rocksdb.cmake

```diff
@@ -442,6 +442,17 @@ else()
       util/crc32c_ppc.c
       util/crc32c_ppc_asm.S)
   endif(CMAKE_SYSTEM_PROCESSOR MATCHES "ppc64")
+  # aarch
+  if(CMAKE_SYSTEM_PROCESSOR MATCHES "aarch64|AARCH64")
+    CHECK_CXX_COMPILER_FLAG("-march=armv8-a+crc+crypto" HAS_ARMV8_CRC)
+    if(HAS_ARMV8_CRC)
+      message(STATUS " HAS_ARMV8_CRC yes")
+      set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -march=armv8-a+crc+crypto -Wno-unused-function")
+      set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -march=armv8-a+crc+crypto -Wno-unused-function")
```

Comment:
We tested that the flag exists in the C++ compiler, but we are also setting it for the C compiler (without checking that the flag is valid for it). I suppose that problems might only arise when using mis-matching C and C++ compiler versions.

Reply:
right .. may be we can get rid of C flag as rocksdb is not using C ?